### PR TITLE
Feat(fe)/new tx history

### PR DESCRIPTION
**Pull request by xnyl**
Firebase-less version of Transaction history. Mint flow changes. Simplified multiple deposits.

### CODE REVIEW

File: src/constants/environmentVariables.ts [hunk 1]
```diff
@@ -18,6 +18,9 @@ const WALLETCONNECT_ENABLED = Boolean(
   process.env.REACT_APP_WALLETCONNECT_ENABLED
 );
 const BSC_MM_ENABLED = Boolean(process.env.REACT_APP_BSC_MM_ENABLED);
+const TX_HISTORY_EXPLORATION = Boolean(
```
Comment (TheAncientGoat) [inline src/constants/environmentVariables.ts:21]: We should remove the FIREBASE env vars

File: src/features/mint/components/MintStatuses.tsx [hunk 2]
```diff
@@ -74,12 +85,12 @@ export const MintDepositToStatus: FunctionComponent<MintDepositToProps> = ({
...
     if (timeRemained > 0) {
       key = showNotification(
-        <AddressValidityMessage
+        <GatewayAddressValidityMessage
           milliseconds={timeRemained}
           destNetwork={getChainConfigByRentxName(tx.destChain).full}
         />,
         {
-          variant: "warning",
+          variant: timeRemained < 6 * 3600 * 1000 ? "error" : "warning",
```
Comment (TheAncientGoat) [inline src/features/mint/components/MintStatuses.tsx:93]: Some sort of time utils to make it clearer that this is 6 hours would be nice

**Review by TheAncientGoat — CHANGES_REQUESTED** [review]

File: src/features/mint/components/MintStatuses.tsx (hunk 2 above)
Comment (xnyl) [reply src/features/mint/components/MintStatuses.tsx]: introduced

File: src/constants/environmentVariables.ts (hunk 1 above)
Comment (xnyl) [reply src/constants/environmentVariables.ts:17]: ok, removed

**Review by TheAncientGoat — APPROVED** [review]

**Pull request merged**